### PR TITLE
fix(docs): change plaintext password to variable form

### PR DESCRIPTION
**Pull request by wangyuancheng01**
**What this PR does / why we need it**:  change plaintext password to variable form

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
=== RUN   TestAccSomethingV0_basic
--- PASS: TestAccSomethingV0_basic (70.75s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       70.796s
```

### CODE REVIEW

File: docs/resources/ddm_schema.md [hunk 1]
```diff
@@ -84,7 +86,7 @@ resource "huaweicloud_ddm_schema" "test"{
   data_nodes {
     id             = huaweicloud_rds_instance.test.id
     admin_user     = "root"
-    admin_password = "test_password_123"
+    admin_password = var.admin_password
```
Comment (ShiChangkuo) [inline docs/resources/ddm_schema.md]: 1. `admin_password` is not defined, it should be `var.password`
2. `password` is not clear， suggest to rename as `rds_password`

File: docs/resources/dms_kafka_instance.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ Manage DMS Kafka instance resources within HuaweiCloud.
 variable "vpc_id" {}
 variable "subnet_id" {}
 variable "security_group_id" {}
+variable "password" {}
```
Comment (ShiChangkuo) [inline docs/resources/dms_kafka_instance.md]:
```suggestion
variable "access_password" {}
```

File: docs/resources/ddm_schema.md (hunk 1 above)
Comment (wangyuancheng01) [reply docs/resources/ddm_schema.md]: It has been fixed.

File: docs/resources/dms_kafka_instance.md (hunk 2 above)
Comment (wangyuancheng01) [reply docs/resources/dms_kafka_instance.md]: ok, I will check others.

File: docs/resources/identity_access_key.md [hunk 3]
```diff
@@ -11,10 +11,12 @@ Manages a permanent Access Key resource within HuaweiCloud IAM service.
 ## Example Usage
 
 ```hcl
+variable "user_1_password" {}
+
 resource "huaweicloud_identity_user" "user_1" {
   name        = "user_1"
   description = "A user"
-  password    = "password123!"
+  password    = user_1_password
```
Comment (ShiChangkuo) [inline docs/resources/identity_access_key.md]: also you can just define it with **password**
```suggestion
  password    = var.password
```

Comment (wangyuancheng01) [reply docs/resources/identity_access_key.md]: done

**Pull request merged**